DynamicSmoothedGUI.py

```diff
@@ -399,9 +399,9 @@ def updateGridSmoothed(self):
             elif self.brokenPathIndex < len(self.brokenPath):
                 lidar_data = self.generate_sensor.generateLidar(
                     degree_freq, self.curr_tile.row, self.curr_tile.col)
-                self.recalc = self.gridEmpty.updateGridLidar(self.curr_tile.x, 
-                    self.curr_tile.y, lidar_data, robot_radius, bloat_factor, 
-                    self.pathSet,self.gridFull)
+                self.recalc = self.gridEmpty.update_grid_tup_data(self.curr_tile.x,
+                                                               self.curr_tile.y, lidar_data, 3, robot_radius, bloat_factor,
```

Comment:
Remember that the argument should be sensor type and it was just 3 for testing purposes